modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+</head>
+
+<body>
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
+			src="http://localhost:18080/documents/d/PRM/liferay_logo_image-png" width="200" />
```

Comment:
I believe `localhost:18080` should be replaced with `[$PORTAL_URL$]` similar to the other templates

Reply:
I had already seen this problem, I ended up not making the substitutions after exporting, I'll send you the changes

Reply:
Fixed